specification/cdn/resource-manager/Microsoft.Cdn/stable/2017-10-12/cdn.json

```diff
@@ -2342,48 +2453,1010 @@
       "properties": {
         "parameters": {
           "description": "Defines the parameters for the condition.",
-          "$ref": "#/definitions/UrlFileExtensionConditionParameters"
+          "$ref": "#/definitions/UrlFileExtensionMatchConditionParameters"
+        }
+      }
+    },
+    "DeliveryRuleUrlFileNameCondition": {
+      "description": "Defines the UrlFileName condition for the delivery rule.",
+      "x-ms-discriminator-value": "UrlFileName",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DeliveryRuleCondition"
+        }
+      ],
+      "required": [
+        "parameters"
+      ],
+      "properties": {
+        "parameters": {
+          "description": "Defines the parameters for the condition.",
+          "$ref": "#/definitions/UrlFileNameMatchConditionParameters"
+        }
+      }
+    },
+    "DeliveryRuleIsMobileCondition": {
+      "description": "Defines the IsMobile condition for the delivery rule.",
+      "x-ms-discriminator-value": "IsMobile",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DeliveryRuleCondition"
+        }
+      ],
+      "required": [
+        "parameters"
+      ],
+      "properties": {
+        "parameters": {
+          "description": "Defines the parameters for the condition.",
+          "$ref": "#/definitions/IsMobileMatchConditionParameters"
+        }
+      }
+    },
+    "DeliveryRuleCookieCondition": {
+      "description": "Defines the Cookie condition for the delivery rule.",
+      "x-ms-discriminator-value": "Cookie",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DeliveryRuleCondition"
+        }
+      ],
+      "required": [
+        "parameters"
+      ],
+      "properties": {
+        "parameters": {
+          "description": "Defines the parameters for the condition.",
+          "$ref": "#/definitions/CookieMatchConditionParameters"
+        }
+      }
+    },
+    "DeliveryRuleUrlWildcardCondition": {
+      "description": "Defines the UrlWildcard condition for the delivery rule.",
+      "x-ms-discriminator-value": "UrlWildcard",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DeliveryRuleCondition"
+        }
+      ],
+      "required": [
+        "parameters"
+      ],
+      "properties": {
+        "parameters": {
+          "description": "Defines the parameters for the condition.",
+          "$ref": "#/definitions/UrlWildcardMatchConditionParameters"
+        }
+      }
+    },
+    "DeliveryRuleUrlSchemeCondition": {
```

Comment:
>DeliveryRuleUrlSchemeCondition [](start = 5, length = 30)

Either move this up after DeliveryRuleIsMobileCondition or update the ordering in the enum

Reply:
reordered